crates/matrix-sdk/src/encryption/mod.rs

```diff
@@ -825,16 +828,61 @@ impl Encryption {
     pub async fn get_user_identity(
         &self,
         user_id: &UserId,
-    ) -> Result<Option<identities::UserIdentity>, CryptoStoreError> {
-        use crate::encryption::identities::UserIdentity;
-
+    ) -> Result<Option<UserIdentity>, CryptoStoreError> {
         let olm = self.client.olm_machine().await;
         let Some(olm) = olm.as_ref() else { return Ok(None) };
         let identity = olm.get_identity(user_id, None).await?;
 
         Ok(identity.map(|i| UserIdentity::new(self.client.clone(), i)))
     }
 
+    /// Get the E2EE identity of a user from the homeserver.
+    ///
+    /// The E2EE identity returned is always guaranteed to be up-to-date. If the
+    /// E2EE identity is not found, it should mean that the user did not set
+    /// up cross-signing.
+    ///
+    /// If you want the E2EE identity of a user without making a request to the
+    /// homeserver, use [`Encryption::get_user_identity()`] instead.
+    ///
+    /// # Arguments
+    ///
+    /// * `user_id` - The ID of the user that the identity belongs to.
+    ///
+    /// Returns a `UserIdentity` if one is found. Returns an error if there was
```

Comment:
Let's linkify this one.

```suggestion
    /// Returns a [`UserIdentity`] if one is found. Returns an error if there was
```

Reply:
Applied.